internal/qtest/test_helper.go

```diff
@@ -194,19 +217,42 @@ func PopulateDatabaseWithInitialData(t *testing.T, db database.Database, sc scm.
 		case pb.TestsRepo:
 			dbRepo.RepoType = pb.Repository_TESTS
 		default:
-			// TODO(Espeland): Could use CreateNamedUser() instead.
-			user = CreateFakeUser(t, db, nxtRemoteID)
+			login := strings.TrimSuffix(dbRepo.Name(), "-labs")
```

Comment:
This can be added as a helper in `ag/repo.go`:

```go
func (t *Repository) User() *User {
	repoName := t.Name()
	login := repoName[:len(repoName)-len(StudentRepoSuffix)]
	return &User{Login: login}
}
```

Alternatively, if it is more useful to return the `login` string, that's fine too.

Reply:
Has been implemented via `ag/repo.go:UserName()`.